File: public/phase-2-20160620022203/C_PLUS_PLUS/notebook-with-numbers.cpp
// Name : Notebook with Numbers 
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char const *argv[]) {
	// input:
	// n : number of sheets
	// k : 
	// a : 
	int index;
	int n, k;

	cin >> n >> k;
	int a[300];

	for(index=0;index<n;index++)
		cin >> a[index];


	// write your code here
	// store your results in `maximum_sum`,`r`

	// output
	// Dummy Data
	int maximum_sum=27, r[300]={3,4,5,8};

	cout << maximum_sum << endl;
	for(index=0;index<(k-1);index++)
		cout << r[index] << " ";

}

File: public/phase-2-20160620022203/C_SHARP/notebook-with-numbers.cs
// Name : Notebook with Numbers 
using System;
namespace Application {
	class notebookwith_numbers {
		public static void Main(string[] args) {
		// input:
		// n : number of sheets
		// k : 
		// a : 
		int index;
		string[] elements = (Console.ReadLine()).Split(' ');
		int n = Convert.ToInt32(elements[0]);
		int k = Convert.ToInt32(elements[1]);
		int[] a = new int[n];
		string[] a_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<n;index++)
			a[index] = Convert.ToInt32(a_elements[index]);


		// write your code here
		// store your results in `maximum_sum`,`r`

		// output
		// Dummy Data
		int maximum_sum=27;
		int[] r = new int[] {3,4,5,8};

		Console.WriteLine(maximum_sum);
		Console.WriteLine(string.Join(" ", r));
		}	
	}
}